app/models/loan.rb

```diff
@@ -1,20 +1,18 @@
 class Loan < ActiveRecord::Base
   include TranslationModule, MediaModule
 
-  belongs_to :cooperative, :foreign_key => 'CooperativeID'
-  belongs_to :division, :foreign_key => 'SourceDivision'
+  belongs_to :division
+  belongs_to :primary_agent, class_name: 'Person'
+  belongs_to :secondary_agent, class_name: 'Person'
+  belongs_to :currency
+  belongs_to :representative, class_name: 'Person'
```

Comment:
I could be mistaken or misremembering, but I think there's a join model that I haven't built yet that's supposed to model that relationship. I double & triple checked the schema document while I was creating all these but I could have somehow missed that one.